apps/docs/docs/how-to-guides/faststore-api/using-graphiql-to-explore-the-faststore-api.mdx

```diff
@@ -5,19 +5,19 @@ sidebar_position: 1
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Using GraphiQL to explore the FastStore API
+# Using a GraphQL IDE to explore the FastStore API
 
-[GraphiQL](https://github.com/graphql/graphiql) is an Integrated Development Environment (IDE) for [GraphQL](https://graphql.org/). You can run GraphiQL on your browser to try out queries and mutations in a GraphQL API.
+[GraphiQL](https://github.com/graphql/graphiql) and [GraphQL Playground](https://github.com/graphql/graphql-playground) are an Integrated Development Environment (IDE) for [GraphQL](https://graphql.org/). You can run them on your browser to try out queries and mutations in a GraphQL API.
 
 You can explore your store's GraphQL data layer by running a local server of your project.
 
 ## Getting started
 
-The first steps to getting GraphiQL up and running are different depending on the framework you are using on your project:
+The first steps to getting them up and running are different depending on the framework you are using on your project:
```

Comment:
Here's more of a suggestion of being more general using "GraphQL IDE" instead of "GraphiQL" because the latter is only used in Gatsby.

Not sure how to best extend this title modification across other pages and references. Is it safe to rename the file and all other occurrences?

<img width="449" alt="CleanShot 2022-08-26 at 14 17 25@2x" src="https://user-images.githubusercontent.com/381395/186958136-9711764b-9273-44f6-bd3d-0bb5856a3ccd.png">

Reply:
I believe it is safe to rename the file and corresponding slugs across other docs.

It would be great if we could also create a URL redirect, in case users have saved the old URL somewhere, but I think we do not have this ability for this portal yet. Do we, @carolinamenezes ?